client/widgets/CGarrisonInt.h

```diff
@@ -81,6 +81,13 @@ class CGarrisonInt :public CIntObject
 	bool checkSelected(const CGarrisonSlot * selected, TQuantity min = 0) const;
 
 public:
+	enum class EGarrisonIntSlotsLayout
+	{
+		ONE_ROW,
+		TWO_ROWS,
+		REVERSED_TWO_ROWS
+	};
```

Comment:
Might be better to move this outside of GarrisonInt (or alternatively - rename to ELayout) - to avoid extremely long `CGarrisonInt::EGarrisonIntSlotsLayout::TWO_ROWS` syntax

Reply:
Also, optionally - integrate smallImgs flag into this enum - since large images is basically just another layout (with larger offsets between images)

Reply:
went for ESlotsLayout to make name more descriptive - check if length satisfies you now